### ISSUE TITLE
Build file is not cleaning up Imports file after build completed

### ISSUE THREAD

**Issue by techthoughts2** [techthoughts2/Catesta#14]
### Expected Behavior

After module build is completed the BuildTask should cleanup leftover artifacts.

### Current Behavior

Once module build is complete the Imports.ps1 is left behind despite having been merged into the psm1.

### Possible Solution

Add a step in the cleanup process to remove the Imports.ps1 file.

### Steps to Reproduce
1. Create a new project
2. Build project
3. Check artifacts folder - Imports.ps1 remains.
